### PR TITLE
prepared release of version 1.6.0

### PR DESCRIPTION
**Pull request by wochinge**
**Proposed changes**:
- prepare 1.6 release

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [x] updated the documentation
- [x] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -17,6 +17,118 @@ This project adheres to `Semantic Versioning`_ starting with version 1.0.
 
 .. towncrier release notes start
 
+[1.6.0] - 2019-12-18
+^^^^^^^^^^^^^^^^^^^^
+
+Deprecations and Removals
+-------------------------
+- `#4935 <https://github.com/rasahq/rasa/issues/4935>`_: Removed ``ner_features`` as a feature name from ``CRFEntityExtractor``, use ``text_dense_features`` instead. If
```
Comment (federicotdn) [inline CHANGELOG.rst]:
```suggestion
- `#4935 <https://github.com/rasahq/rasa/issues/4935>`_: Removed ``ner_features`` as a feature name from ``CRFEntityExtractor``, use ``text_dense_features`` instead.
```

**Comment by wochinge** [comment]
thanks for the catch @federicotdn

**Comment by wochinge** [comment]
@federicotdn noooooo 😀

**Comment by federicotdn** [comment]
Sorry!!!!!! Merged it without thinking 😭

**Comment by tmbo** [comment]
Merged (as the previous build passed and no sig changes)

**Comment by wochinge** [comment]
@tmbo we still have to bump up the rasa-sdk version as soon as 1.6 is released.

**Pull request merged**